### PR TITLE
remove user-progress.scss contents

### PR DESCRIPTION
**Pull request by davidsbailey**
pulling the threads from https://github.com/code-dot-org/code-dot-org/pull/34477 and also https://github.com/code-dot-org/code-dot-org/pull/34470, I found some more CSS to remove.

### CODE REVIEW

File: shared/css/user-progress.scss [hunk 1]
```diff
@@ -1,154 +0,0 @@
...
-  background-color: $level_review_accepted;
-  color: white !important;
-}
-
-.level_link.review_rejected, .level_link.review_rejected:link, .level_link.review_rejected:visited {
-  background-color: $level_review_rejected;
-  color: white !important;
-}
-
-.user-stats-block
```
Comment (davidsbailey) [inline shared/css/user-progress.scss:78]: this is the only one that I could find which was actually getting applied to any elements in our product. I checked the script overview page and the MiniView, and they appeared to render the same with or without these styles.

File: shared/css/user-progress.scss [hunk 2]
```diff
@@ -1,154 +1,4 @@
...
-  dd {
-    line-height: 36px;
-    margin-left: 0;
-    margin-right: 25px;
-  }
-}
+/* 
+ * This file is no longer needed, however removing it causes the drone run to fail.
+ * It should be removed once we figure out how to do so without disrupting drone.
+ */
```
Comment (Erin007) [inline shared/css/user-progress.scss:4]: hmm - interesting.

**Review by Erin007 — APPROVED** [review]
thanks for catching all these little tendrils! I'm curious why Drone is expecting this file to exist, but looks good as is.

**Comment by davidsbailey** [comment]
> thanks for catching all these little tendrils! I'm curious why Drone is expecting this file to exist, but looks good as is.

I am too! See discussion in [#test-infrastructure](https://codedotorg.slack.com/archives/C2ALWLRHN/p1588874466018000) . Hopefully merging this PR will help simplify things when we go to try and remove the file entirely without breaking drone.

**Pull request merged**